esp/public/media/default_styles/forms.css

```diff
@@ -341,3 +341,11 @@ div.raw_id_admin {
 div.calendar {
     z-index: 10; /* 2 actually suffices, but I'm leaving a safety margin. -ageng 2009-02-02 */
 }
+
+.form_error {
```

Comment:
Hm, what does this change?

Reply:
It changes the "This field is required." error message format so it stands out more from the field labels

Reply:
Thanks for mentioning that. It works for circles, but barebones and bigpicture only load thme_compiled.css, which gets clobbered. How should I go about addressing that? It feels wrong to put it in elements/html or main.html

Reply:
If you add it to one of the css files that the theme includes, it will end up in theme_compiled.css.  I any `less` files in `esp/public/media/theme_editor/less` will work?  Looks like there's a `forms.less` you could add it to.  But test it, because I don't remember exactly how all this stuff works.